src/index.css

```diff
@@ -0,0 +1,46 @@
+@import url("https://pixeldesign.netlify.app/pixel-design.css");
+@import url("https://fonts.googleapis.com/css2?family=Kodchasan:wght@200;300;400;500;600;700&family=Montserrat:wght@200;300;400;500;600;700;800;900&family=Work+Sans:wght@200;300;400;500;600;700&display=swap");
+
+body {
+  margin: 0;
+  /* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif; */
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+body::-webkit-scrollbar {
+  display: none;
+}
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+:root {
+  font-size: 62.5%;
+  font-family: "Montserrat", sans-serif;
+}
+html {
+  --btn-yellow: #ff9f00;
+  --btn-orange: #f08804;
+  --bg-color-light: #cccccc;
+  --index--1: -1;
+  --index-9: 9;
+  --index-99: 99;
+  --index-999: 999;
+  scroll-behavior: smooth;
```

Comment:
Suggestion: Don't use values with this much high difference as this could even work with 1,2,3 and 4 values. if it is in order then it's easy to manage.

Reply:
True, I will take care of it next time!